app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -161,6 +161,7 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
         String message;
```

Comment:
please remove edits not related with your PR

Reply:
Thanks dear i have removed the space